<!--
  The turtlebot people (or whatever) follower nodelet.   
 -->
<launch>
  
  <include file="$(find lilee_follower)/launch/includes/velocity_smoother.launch.xml"/>

  <param name="camera/rgb/image_color/compressed/jpeg_quality" value="22"/>

<!--
  <include file="$(find turtlebot_bringup)/launch/3dsensor.launch">
    <arg name="rgb_processing"                  value="true"/>
    <arg name="depth_processing"                value="true"/>
    <arg name="depth_registered_processing"     value="false"/>
    <arg name="depth_registration"              value="false"/>
    <arg name="disparity_processing"            value="false"/>
    <arg name="disparity_registered_processing" value="false"/>
    <arg name="scan_processing"                 value="false"/>
  </include> 

  <node pkg="topic_tools" type="throttle" name="camera_throttle"
        args="messages camera/rgb/image_color/compressed 5"/>
-->
  <!--  Load turtlebot follower into the 3d sensors nodelet manager to avoid pointcloud serializing -->
  <node pkg="nodelet" type="nodelet" name="lilee_follower"
	args="load lilee_follower/LileeFollower /image_proc_nodelet">
    <remap from="lilee_follower/cmd_vel" to="follower_velocity_smoother/raw_cmd_vel"/>
    <remap from="depth/points" to="camera/depth/points"/>
    <param name="enabled" value="true" />
    <param name="x_scale" value="7.0" />
    <param name="z_scale" value="2.0" />
    <param name="min_x" value="-0.5" />
    <param name="max_x" value="0.5" />
    <param name="min_y" value="-0.37" />
    <param name="max_y" value="0.5" />
    <param name="max_z" value="10" />
    <param name="goal_z" value="0.3" />
  </node>

</launch>
